crates/bevy_ecs/src/schedule_v3/schedule.rs

```diff
@@ -268,10 +286,23 @@ impl Dag {
             topsort: Vec::new(),
         }
     }
+
+    /// A directed graph.
+    pub fn graph(&self) -> &DiGraphMap<NodeId, ()> {
+        &self.graph
+    }
+
+    /// A cached topological ordering of the graph.
+    pub fn topsort(&self) -> &[NodeId] {
```

Comment:
IMO this would be clearer as `cached_topsort`, as it's easy to assume that this is generated when the method is called.